create_graph.py

```diff
@@ -11,19 +11,34 @@ def process_pkl_files(triples_file,labels_file):
     triples_df = pd.read_csv(triples_file,sep = '	')
     triples_df.columns.str.lower()
 
-    labels = pd.read_csv(labels_file, sep = '	')
-    labels.columns.str.lower()
-
-    return triples_df,labels
+    #labels = pd.read_csv(labels_file, sep = '	')
+    #labels.columns.str.lower()
+    
+    
+    # In order to show all the labels available in the labels_file tab deliminated txt file, read the file with pandas fixed-width lines reader
+    # (this will destroy the frame of the file though!!!)
+    labels_fwf = pd.read_fwf(pkl_microbiome_labels_file)
+    print("The total number of labels is", len(all_labels))
```

Comment:
should the line 21 variable be labels_fwf? and should the line 20 variable be labels_file, not pkl_microbiome_labels_file?

Reply:
right, sorry about that, It's the old script that I am running, I will download the new github zip instead!